CarRecognition/Source Files/Common/Models/RecognitionResult.swift

```diff
@@ -45,9 +45,9 @@ internal struct RecognitionResult: CustomStringConvertible {
         self.label = label
         if let car = carsDataService.map(classifierLabel: label) {
             recognition = .car(car)
-        } else if label == "other car" {
+        } else if label == "other_car" {
```

Comment:
You move the second string to some constant, why not go with this one too? :)

Reply:
Will do 👍